dist/index.d.ts

```diff
@@ -54,8 +56,24 @@ interface IPosisProcessRegistry {
 	// if your bundle consists of several programs you can pretend that we have a VFS: "ANI/MyBundle/BundledProgram1"
 	register(imageName: string, constructor: new (context: IPosisProcessContext) => IPosisProcess): boolean;
 }
+interface IPosisCooperativeSchedulingV1 {
```

Comment:
Can we just remove V1? A stub implementation of yield is trivial. An empty function fulfills the requirement.